.github/workflows/test.yml

```diff
@@ -28,16 +28,15 @@ jobs:
     name: ${{ matrix.platform }} py${{ matrix.python-version }}
     runs-on: ${{ matrix.platform }}
     strategy:
+      fail-fast: true
       matrix:
-        # Run all supported Python versions on linux
-        python-version: ["3.8", "3.9", "3.10"]
         platform: [ubuntu-latest]
-        # Include one windows and macos run
+        python-version: [3.8, 3.9, "3.10"]
         include:
-        - platform: macos-latest
-          python-version: "3.9"
-        - platform: windows-latest
-          python-version: "3.9"
+          - platform: macos-latest
+            python-version: 3.9
+          - platform: windows-latest
+            python-version: 3.9
```

Comment:
Do we want `3.10` to be the default still?

Reply:
I've updated it to use 3.10 instead